packages/react-resize-handle/src/hooks/useKeyboardHandler.ts

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import { GrowDirection, SupportedKeys } from '../types';
 
 export type UseKeyboardHandlerOptions = {
-  onValueChange: (value: number) => void;
+  onValueChange: (value: number, event: KeyboardEvent) => void;
```

Comment:
Let's rather break it and use `EventHandler` type like in Fluent to avoid confusion. It also will be more explicit about interactions.

Reply:
I started working on exactly this before the weekend :)